src/browsers.c

```diff
@@ -132,6 +132,9 @@ static const char *browsers[][2] = {
   {"libmpv", "Others"},
   {"aria2", "Others"},
 
+  /* WordPress Cron*/
+  {"WordPress/","Cron"},
```

Comment:
It would be better to have a space after comma, please check your following lines as well.

```suggestion
  {"WordPress/", "Cron"},
```